src/time_zone_lookup.cc

```diff
@@ -207,8 +212,9 @@ time_zone local_time_zone() {
   char* localtime_env = nullptr;
   if (strcmp(zone, "localtime") == 0) {
 #if defined(_MSC_VER)
-    // System-specific default is just "localtime".
-    _dupenv_s(&localtime_env, nullptr, "LOCALTIME");
+    TIME_ZONE_INFORMATION zi = {};
+    GetTimeZoneInformation(&zi);
+    return fixed_time_zone(seconds(-zi.Bias * 60));
```

Comment:
I think this is largely antithetical to CCTZ's goal of providing accurate timezone information for a region.  Indeed, if I had my time again I would omit `fixed_time_zone()` from the API.  It is typically more a part of a problem rather than a solution.